src/models/parameter.ts

```diff
@@ -1,41 +1,63 @@
 import { Record as ImmuRecord } from "immutable";
-import { JsonMap } from "../lib/types";
+import { OSCQueryRNBOInstanceParameter } from "../lib/types";
 
-export class ParameterRecord extends ImmuRecord({
+export type ParameterRecordProps = {
+	min: number;
+	max: number;
+	name: string;
+	normalizedValue: number;
+	path: string;
+	type: string;
+	value: number;
+}
+export class ParameterRecord extends ImmuRecord<ParameterRecordProps>({
 
-	name: "name",
-	value: 0,
 	min: 0,
 	max: 1,
+	name: "name",
+	normalizedValue: 0,
+	path: "",
 	type: "f",
-	normalizedValue: 0
+	value: 0
 
 }) {
 
-	public static arrayFromDescription(desc: JsonMap, prefix?: string): ParameterRecord[] {
+	public static fromDescription(name: string, desc: OSCQueryRNBOInstanceParameter): ParameterRecord {
+		return new ParameterRecord({
+			min: desc.RANGE?.[0]?.MIN,
+			max: desc.RANGE?.[0]?.MAX,
+			name,
+			normalizedValue: desc.CONTENTS.normalized.VALUE,
+			path: desc.FULL_PATH,
+			type: desc.TYPE,
+			value: desc.VALUE
+		});
+	}
```

Comment:
I suspect that something is wrong here with polyphonic patcher parameters

Reply:
yeah the feedback synth failed due to that. should be fixed now